### PR TITLE
Only show reading time if more than 4 minutes

### LINKED ISSUE #282: Do not show reading time if is less than 1 or 2 min
It does not make sense to show it for small few line articles.

This should be configurable.

### PR DESCRIPTION
**Pull request by iranzo**
Closes #282

### CODE REVIEW

**Comment by talha131** [comment]
Thank you @iranzo. We should make the minimum time configurable and set it to a default value.

File: templates/_includes/post_stats.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% if article.stats and article.stats['read_mins'] %}
+{% if article.stats and article.stats['read_mins'] and article.stats['read_mins'] > 2 %}
```
Comment (talha131) [inline templates/_includes/post_stats.html]: This minimum time has to be configurable. And set to a default value.

**Comment by iranzo** [comment]
@talha131 how do we want to name that variable?

**Comment by talha131** [comment]
How about `READING_TIME_LOWER_LIMIT` and a default value of 4 min?

**Comment by iranzo** [comment]
Done in last push

File: templates/_includes/post_stats.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{% if article.stats and article.stats['read_mins'] and article.stats['read_mins'] > 2 %}
+{% if article.stats and article.stats['read_mins'] and article.stats['read_mins'] > READING_TIME_LOWER_LIMIT|default(4, true) %}
```
[the remark this answers is not in the document]
Comment (iranzo) [reply templates/_includes/post_stats.html]: Other templates had it in the template itself:

~~~
     {% for tag in article.tags|sort %}
                <li><a href="{{ SITEURL }}/{{ TAGS_URL|default('tags') }}#{{ tag.slug }}-ref">{{ tag }}
                    {% for aTag, tagged_articles in tags if aTag == tag %}
                    <span>{{ tagged_articles|count }}</span>
~~~

(for example)

I've added a new push that uses _defaults for it

**Comment by iranzo** [comment]
Change added to changelog and rebased onto 'next' (I tought this was getting into actual branch)

**Comment by iranzo** [comment]
@talha131 anything missing to address here?

File: templates/base.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
         {% else %}
         <meta name="author" content="{{ AUTHOR }}" />
         {% endif %}
-
+        {% from '_includes/_defaults.html' import READING_TIME_LOWER_LIMIT with context %}
```
Comment (talha131) [inline templates/base.html]: Why import variable from default here? It should be imported in the post_stats file; right above where it is used.

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,6 +19,11 @@ Version 2.0 (under development)
   it easier for you to customize or even *localize* the theme
 * Adds author blurbs at the end of the article
 
+To Be Released
```
Comment (talha131) [inline CHANGELOG.md]: Please move this patch to the top of the file.

I know 2.1 and 2.1.1 notes are not present and (under development) heading is also misleading. I need to fix that.

File: templates/base.html (hunk 3 above)
Comment (iranzo) [reply templates/base.html]: Changed in next push, can you recheck?

File: CHANGELOG.md (hunk 4 above)
Comment (iranzo) [reply CHANGELOG.md]: Changed in next push

File: templates/base.html [hunk 5]
```diff
@@ -17,7 +17,6 @@
         {% else %}
         <meta name="author" content="{{ AUTHOR }}" />
         {% endif %}
-
```
Comment (talha131) [inline templates/base.html]: This file should not be in the PR at all. 😄

Comment (iranzo) [reply templates/base.html]: Done!

**Comment by talha131** [comment]
Thank you @iranzo

**Pull request merged**